app/lib/actions/katello/content_view/errata_mail.rb

```diff
@@ -11,7 +11,7 @@ def run
 
           content_view = ::Katello::ContentView.find(input[:content_view])
           environment = ::Katello::KTEnvironment.find(input[:environment])
-          users = ::User.select { |user| user.receives?(:promote_errata) && user.can?(:view_content_views, content_view) }
+          users = ::User.select { |user| user.receives?(:promote_errata) && user.organizations.include?(content_view.organization) && user.can?(:view_content_views, content_view) }
```

Comment:
This could be a bit more efficient with `user.organization_ids.include?(content_view.organization_id)`.